### PR TITLE
Added support for new competing activity type and removed changing status from on_ready

### PR DESCRIPTION
**Pull request by Soheab**
New activity type can be found [here](https://discordpy.readthedocs.io/en/latest/api.html#discord.ActivityType.competing)

About changing status in on_ready: this is not recommended and can cause issues. See more in the `?tag orp` tag on the [discord.py server](https://discord.gg/dpy) I moved it to data.py to use the `activity` and `status` kwargs to do the same thing.

Added owner ids from config as the owners for the bot.

Changed playing and playing_type to activity and activity_type because this confused some people.

Also added a comment in index.py about intents so users can learn more about them.

### CODE REVIEW

File: utils/data.py [hunk 1]
```diff
@@ -1,12 +1,36 @@
...
+        # Check if user desires to have something other than online
+        if config.status_type.lower() == "idle":
+            status_type = discord.Status.idle
+        elif config.status_type.lower() in ("do not disturb", "dnd"):
+            status_type = discord.Status.dnd
+        else:
+            status_type = discord.Status.online
+
+        # Check if user desires to have a different type of activity
+        if config.activity_type.lower() == "listening":
```
Comment (AlexFlipnote) [inline utils/data.py:23]: Now that I see it, maybe make **config.activity_type.lower()** to a variable to then compare, makes it much cleaner considering I convert it to .lower() constantly.

File: utils/data.py [hunk 2]
```diff
@@ -23,13 +47,13 @@ def get_destination(self, no_pm: bool = False):
             return self.context.author
 
     async def send_error_message(self, error):
-        destination = self.get_destination(no_pm=True)
+        destination = self.get_destination(no_pm = True)
```
Comment (AlexFlipnote) [inline utils/data.py:50]: Having space between equals makes my linter (flake8) slap my face for being bad code, pls fix yes.

**Review by AlexFlipnote — COMMENTED** [review]
Please fix, the equals one is for all changes

File: utils/data.py (hunk 1 above)
Comment (Soheab) [reply utils/data.py]: Yes that's better.

File: config.json.example [hunk 3]
```diff
@@ -5,10 +5,11 @@
     86477779717066752
   ],
   "prefix": [
-    "!"
+    "!",
+    "?"
```
Comment (AlexFlipnote) [inline config.json.example:9]: Wait, why is there a question mark here?

Comment (Soheab) [reply config.json.example:9]: More common prefixes:)

Comment (Soheab) [reply config.json.example]: Removed it

**Pull request merged**